docs/toc.md

```diff
@@ -5,7 +5,7 @@
 ## [Getting Started with .NET](standard/getting-started.md)
 ## [Tour of .NET](standard/tour.md)
 ## [.NET Architectural Components](standard/components.md)
-## [.NET Standard Library](standard/library.md)
+## [.NET Standard](standard/library.md)
```

Comment:
And also the file name can be `standard/`
1. `info.md`
2. `index.md`
3. `net-standard.md` (confusion with path name)

for me I'll go with `2`